src/chalkjson.nim

```diff
@@ -442,12 +442,12 @@ proc extractOneChalkJson*(chalkData: string, path: string): ChalkDict =
 
 # Output ordering for keys.
 proc orderKeys*(dict: ChalkDict,
-                tplate: MarkTemplate | ReportTemplate): seq[string] =
+                tplate: AttrScope): seq[string] =
   var tmp: seq[(int, string)] = @[]
   for k, _ in dict:
-    var order = chalkConfig.keySpecs[k].normalizedOrder
-    if tplate != nil and k in tplate.keys:
-      let orderOpt = tplate.keys[k].order
+    var order = get[int](getChalkScope(), "keyspec." & k & ".normalized_order")
```

Comment:
is this guaranteed to be present?

Reply:
looks like it is https://github.com/crashappsec/chalk/blob/6b6c9c260a9c18cc46f25407b773da83e9dcd80f/src/configs/chalk.c42spec#L827-L829